src/main/java/objects/quests/QuestStep.java

```diff
@@ -3,23 +3,24 @@
 import constants.Constant;
 import constants.Skills;
 import objects.Item;
+import objects.ItemStack;
```

Comment:
Done in different commit since you last pulled. #26 
In future make use of the issue (in this case #25 and comment or show you're working on it. Prevents duplication and waste of effort in places.)